docs.bruno/ASAP - Testing.bru

```diff
@@ -0,0 +1,11 @@
+meta {
```

Comment:
Consider adding comments to explain the purpose of the `meta` and `get` blocks in the file. This will make the code more maintainable and easier for others to understand. [medium]